docs/application/web/guides/machine-learning/trainer.md

```diff
@@ -0,0 +1,294 @@
+kernel_size = "3, 3"
+activation = relu
+
+[layer_pooling2d2]
+type = pooling2d
+pooling = max
+pool_size = "2, 2"
+
+[layer_flatten3]
+type = flatten
+
+[layer_fc4]
+type = fully_connected
+unit = 100
+activation = relu
+
+[layer_fc5]
+type = fully_connected
+unit = 10
+activation = softmax
+```
+
+To load the configuration use `createModel` function.
+
+```javascript
+var configFile = "documents/model.ini";
+var model = tizen.ml.trainer.createModel(configFile);
+```
+
+Compile the model.
+
+```javascript
+model.compile({
+    loss: "cross",
+});
+```
+
+## Training
+
+Connect the dataset to the model.
+
+```javascript
+model.setDataset(dataset);
+```
+
+Now the model is ready to train. To do that use `run` method.
+
+```javascript
+var options = {
+    epochs: 2,
+    batch_size: 32,
+};
+
+function successCallback() {
+    console.log("done");
+}
+
+function errorCallback(error) {
+    console.log(error);
+}
+
+model.run(options, successCallback, errorCallback);
+```
+
+## Saving trained model to a file
+
+The trained model can be saved to a file for the future use. Let's use the `FORMAT_INI_WITH_BIN` format, which will produce two files: an `ini` file with the network structure and a binary file with the weights. There are more formats which can be found in [API reference](https://docs.tizen.org/application/web/api/7.0/device_api/mobile/tizen/ml_trainer.html#Model::saveToFile). Remember to add a proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites).
+
+```javascript
+var modelFile = "documents/trained_model.ini";
+model.saveToFile(modelFile, "FORMAT_INI_WITH_BIN");
+```
+
+## Loading trained model from a file
+
+A trained model can be loaded from a file, both its structure and weights. If the model was saved using `FORMAT_INI_WITH_BIN` format, which produces two files, provide a path to the `ini` file. It contains a reference to the binary file with weights. It will be loaded during the compilation phase. Remember to add a proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites).
+
+```javascript
+var model = tizen.ml.trainer.createModel();
+var modelFile = "documents/trained_model.ini";
+model.load(modelFile, "FORMAT_INI_WITH_BIN");
+model.compile();
+```
+
+## Destroying the model
+
+If the model or its elements are not used anymore, they should be destroyed to avoid memory leaks. Disposing a model also destroys all objects associated with it (such as layers).
+
+```javascript
+model.dispose();
+```
+
+## Using the trained model for inference
+
+The trained model can be loaded using either [ML Single API]() or [ML Pipeline API](). This is an example for the first one:
+
+```javascript
+var single = tizen.ml.single.openModel("documents/trained_model.ini", null, null, "NNTR_INF", "AUTO");
+
+// ...
+
+single.close();
+```
+
+## Related information
+
+- Dependencies
+  - Tizen 7.0 and Higher for Mobile
+  - Tizen 7.0 and Higher for Wearable
+  - Tizen 7.0 and Higher for TV
```

Comment:
Should there also be a separate declaration to distinguish the version that is required for IoT?
For example,
Tizen 7.0 and Higher for IoT

Reply:
It is good now. There is no documentation of Web API for IoT devices.